### DISCUSSION TITLE
大概什么时候能发布正式版？

### DISCUSSION THREAD

**Discussion by mlkt** [element-plus/element-plus#2872] [Q&A] [answered]
我看commit最近每天都有很多大改动和重构，我自己用也遇到了一些大大小小的bug，文档里的教程也有滞后，有些功能我看了源代码才知道。不知贵公司的项目进度是怎么规划的，大概啥时候可以有正式版？没有催的意思，就是想了解一下

**Comment by iamkun** [comment]
There's no timetable ATM.
We will make a stable release as faster as we can.

Feel free to ask for any help here if needed, and welcome to join us to make our document better.

**Reply by ac51300696** [reply]
I want join to fix some bug and pr some feature，hava any  development document or chart？

**Comment by sxzz** [comment, marked as answer]
Please reference https://github.com/element-plus/element-plus/discussions/3065
